src/_posts/2019-10-11-elastic-stack-introduction.md

```diff
@@ -0,0 +1,915 @@
+---
+layout: post
+asset-type: post
+name: elastic-stack-introduction
+title: Elastic Stack Introduction
+date: 2019-10-11 00:00:00 +00:00
+author: André Guelfi Torres
+description: An introduction on how to use the Elastic Stack to monitor your applications
+image:
+ src: /assets/custom/img/blog/2019-10-11-elastic-stack-introduction/monitor.jpg
+tags:
+- monitoring
+- logging
+---
+
+# Introduction
+
+But when the software that you are running is critical for your company you can't have logs just for analysing things that happened some time ago, having the client tells you that your application is broken while you didn't even know what was happening is a real problem, and one of the ways to deal with that is using monitoring and logging. 
```

Comment:
Why does this sentence start with `But` ? It seems to follow a previous statement but is the first sentence of the post.